docs/reference/changelog.rst

```diff
@@ -0,0 +1,92 @@
+*********
+Changelog
+*********
+
+3.0.0 (2024-Jun-28)
+-------------------
+
+Craft Application 3.0.0 implements the ``BuildPlanner`` class and can create
+a build plan. This is a breaking change because it requires more fields to
+be defined.
+
+.. warning::
+
+   ``platforms`` is now a required field in the ``Project``
+
+   ``platforms``, ``base``, and ``build-base`` are now required fields in the
+   ``BuildPlanner`` model
+
+Application
+===========
+
+* Extends ``add_command_groups()`` to accept a sequence instead of a list.
+* Adds support for building architecture-independent artefacts by accepting
+  ``all`` as the ``build-for`` target.
+
+Models
+======
+
+* Adds a default ``Platform`` model. See :doc:`platforms</reference/platforms>`
+  for a reference of the model.
+* Adds a default ``get_build_plan()`` function to the ``BuildPlanner`` class.
+  See :doc:`Build plans</explanation/build-plans>` for an explanation of how
+  the default ``get_build_plan()`` works.
+* Changes ``BuildPlanner`` from an abstract class to a fully implemented class.
+  Applications can now use the ``BuildPlanner`` class directly.
+
+For a complete list of commits, check out the `2.9.0`_ release on GitHub.
+
+2.8.0 (2024-Jun-03)
+-------------------
+
+Commands
+========
+
+* Fixes a bug where the pack command could accept a list of parts as command
+  line arguments.
+* Adds support for commands to accept multiple ``platform`` or ``build-for``
+  values from the command line as comma-separated values.
+
+Remote build
+============
+
+* Retries more API calls to Launchpad.
+* Adds an exponentially backoff to API retries with a maximum total delay of
+  62 seconds.
```

Comment:
Is 62 seconds really the correct value?

Reply:
It does seem odd but yes: 2+4+8+16+32 = 62 seconds

https://github.com/canonical/craft-application/commit/b97d76d62d0c48ce216db37376353401508895c4